examples/tx-example.py

```diff
@@ -177,6 +180,18 @@ def get_account_assets():
         print('Asset id = {}, balance = {}'.format(
             asset.asset_id, asset.balance))
 
+@trace 
+def query_transactions(first_time=None, last_time=None, first_height=None, last_height=None):
+    query = iroha.query('GetAccountTransactions', account_id='admin@test',
```

Comment:
If we are signing with `ADMIN_PRIVATE_KEY` I suggest to use the const as user:

```suggestion
    query = iroha.query('GetAccountTransactions', account_id=ADMIN_ACCOUNT_ID,
```